open-company/open-company-storage
# Strip script tags to prevent xss.

This change removes tags that are problems when displaying in a web browser.

To test:

Use postman to make a request to the storage service.
- copy the header information from a succesful post creation (Content-Type, Authorization, etc)
- copy the body from the succesful post
- in postman use the same body but add a `<script> alert('wtf'); </script>` or `<style>lfjd</style>` tag in the body or headline.
- [x] the returned request should show the tags removd
- [x] query the database to see that the tags are not stored

- Repeat the same steps for board creation
- change the name to include the `<script>` or `<style>` tags
- [x] are the tags removed?

- Repeat the same steps for invites 
- add a note field with the tags
- [x] are they removed?
